server/server.js

```diff
@@ -33,8 +38,25 @@ console.log('create session')
  *  this endpoint closes session
  */
 server.get('/close', (req, res) => {
```

Comment:
Might be you forgot to add URL parameter as first.  `server.get('/close/:closeId', ...`
Also use are usign `sessionID` name, are you sure that you use `closeId`, not `sessionId`?